### PR TITLE
Update build_pushprox.yml

### PR DESCRIPTION
**Pull request by rajeshtessell**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

Dependent Services #
- [ ] Governance
- [ ] IAM
- [ ] Deployer
- [ ] DB Systems
- [ ] Tenant
- [ ] Software Library
- [ ] Backup
- [ ] Conductor
- [ ] Monitoring
- [ ] DMM

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Environment**:

- [ ] Dev
- [ ] Staging

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .github/workflows/build_pushprox.yml [hunk 1]
```diff
@@ -45,12 +50,25 @@ jobs:
...
         run: |
           echo "Building the service"
           git config --global url."https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com".insteadOf "https://github.com"
           make service
           make image
+          
+      - name: Building Tar files for pushprox
+        run: |
+          for os in ${OS_NAMES[@]};do
+          make os=$os pushprox-client
```
Comment (sundeeptessell) [inline .github/workflows/build_pushprox.yml:65]: Add Indentation.

File: .github/workflows/build_pushprox.yml [hunk 2]
```diff
@@ -154,6 +172,24 @@ jobs:
         run: |
           echo "DOCKERHUB_ORG=tessellprod" >> $GITHUB_ENV
           
+          
+      - name: Pushing generated tar files to nexus
+        if: ${{ startsWith(env.SOURCE_BRANCH,'rel-') || env.SOURCE_BRANCH == 'main' }}
+        run: |
+            for os in ${OS_NAMES[@]};do
+            mkdir -p $HOME/.m2
```
Comment (sundeeptessell) [inline .github/workflows/build_pushprox.yml:180]: Add indentation.

**Review by sundeeptessell — CHANGES_REQUESTED** [review]

File: .github/workflows/build_pushprox.yml (hunk 1 above)
Comment (rajeshtessell) [reply .github/workflows/build_pushprox.yml]: added

File: .github/workflows/build_pushprox.yml (hunk 2 above)
Comment (rajeshtessell) [reply .github/workflows/build_pushprox.yml]: added

**Pull request merged**